external/mbedtls/ssl_tls.c

```diff
@@ -7001,12 +7027,14 @@ int mbedtls_ssl_handshake_step( mbedtls_ssl_context *ssl )
         return( MBEDTLS_ERR_SSL_BAD_INPUT_DATA );
 
 #if defined(MBEDTLS_SSL_CLI_C)
-    if( ssl->conf->endpoint == MBEDTLS_SSL_IS_CLIENT )
+    if( ssl->conf->endpoint == MBEDTLS_SSL_IS_CLIENT ){
         ret = mbedtls_ssl_handshake_client_step( ssl );
+    }
 #endif
 #if defined(MBEDTLS_SSL_SRV_C)
-    if( ssl->conf->endpoint == MBEDTLS_SSL_IS_SERVER )
+    if( ssl->conf->endpoint == MBEDTLS_SSL_IS_SERVER ){
```

Comment:
Because this is in external, you don't need to follow TizenRT coding rules. But let follow mbedtls itself coding rules. You mix two ways for if conditional.
You do below at line above
```
if ( )
{
}
```
And you do below here
```
if ( ) {
}
```